stingray/tests/test_crossspectrum.py

```diff
@@ -322,19 +322,21 @@ def test_different_dt(self):
 
     def test_different_tseg(self):
         time2 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
-        counts2_test = np.random.poisson(0.01, size=len(time2))
+        counts2_test = np.random.poisson(1000, size=len(time2))
```

Comment:
What was wrong with the old Poisson rate? :)

Reply:
I had a failure for low count rate in a test. Randomly... :)